astroid/scoped_nodes.py

```diff
@@ -2886,9 +2886,9 @@ def slots(self):
         :rtype: list(str) or None
         """
 
-        def grouped_slots():
+        def grouped_slots(mro):
```

Comment:
```suggestion
        def grouped_slots(mro: ?) -> None:
```
Could you add typing on modified functions, please ? :)